tools/pylint-extensions/pylint-guidelines-checker/pylint_guidelines_checker.py

```diff
@@ -372,14 +374,19 @@ def visit_functiondef(self, node):
         :return: None
         """
         try:
-            if node.parent.name.endswith("Client") and node.is_method() and not node.name.startswith("_") and \
-                    node.parent.name not in self.ignore_clients:
-                if node.args.kwarg and "azure.core.tracing.decorator.distributed_trace" not in node.decoratornames() \
-                        and "builtins.classmethod" not in node.decoratornames():
-                    self.add_message(
-                        msgid="client-method-missing-tracing-decorator", node=node, confidence=None
-                    )
-        except AttributeError:
+            path = node.root().name
```

Comment:
(un-related line) Do you think maybe in the description of the error we should say something about how this only applies to methods that make network calls? The scenario I'm thinking of is when we have a false positive and someone reads the error and adds it because the rule says so